<launch>	

    <include file="$(find openni2_launch)/launch/openni2.launch" />

    <include file="$(find moveit_generated)/launch/wm_sensor_manager.launch" />
	<include file="$(find moveit_generated)/launch/wm_move_group.launch" />
	
	<node name="wm_arm_driver_node" pkg="wm_arm_driver" type="wm_arm_driver_node" output="screen" >

		<param name="urdfParam" value="robot_description" />
		<param name="move_group_name" value="arm" />

		<param name="maxVelocity_joints123" value="30" />
		<param name="maxVelocity_joints456" value="30" />
		<param name="jointMovement_synchronization" value="1" />

		<param name="maxTorque_joint1" value="5" />
		<param name="maxTorque_joint2" value="5" />
		<param name="maxTorque_joint3" value="5" />
		<param name="maxTorque_joint4" value="5" />
		<param name="maxTorque_joint5" value="5" />
		<param name="maxTorque_joint6" value="3" />

		<param name="checkTorque_rate" value="20" />
		<param name="checkPosition_rate" value="20" />
		<param name="checkVelocity_rate" value="20" />

	</node>
</launch>
